modules/nf-core/fgbio/fastqtobam/tests/bam.config

```diff
@@ -0,0 +1,3 @@
+process {
+    ext.suffix = "bam"
+}
```

Comment:
is this needed? it's the default value anyway

Reply:
Not really, but I took this from the old tests where they tested it too. Do you want me to remove it?